app/src/main/res/xml/appearance_settings.xml

```diff
@@ -41,6 +41,13 @@
         app:singleLineTitle="false"
         app:iconSpaceReserved="false" />
 
+    <SwitchPreference
```

Comment:
If we remove the setting, then we should have a tooltip show up when the thumbnail is long-pressed so that the user doesn't get confused and think that the menu isn't working.

Something like "Long-press thumbnail to preview video".

Maybe add a visible service-coloured border to the thumbnail as well when the preview is playing to visually highlight it a little.

Also, I don't know what the current status is, but from the Dropbox video in the OP, I it seems that the preview keeps playing forever. I think if it isn't implemented yet, then there should be an obvious way to stop the preview. Maybe by tapping the thumbnail again, or by tapping the list item in general, or tapping elsewhere, or scrolling it out of the screen. Something to make it stop when you're done.

I'm assuming a standard use case will be people browsing their feed and previewing many many videos to decide what to watch. Does playing another preview cause the current one to stop? If not, you could have something like 5 previews playing at the same time, which will be messy.

Reply:
> If we remove the setting, then we should have a tooltip show up when the thumbnail is long-pressed so that the user doesn't get confused and think that the menu isn't working.

It would be useful regardless of whether we remove the setting or not

> it seems that the preview keeps playing forever.

No, it stops after the frames are finished

> Does playing another preview cause the current one to stop?

Yes